rio_tiler_pds/landsat/aws/landsat_collection2.py

```diff
@@ -0,0 +1,210 @@
+"""AWS Landsat Collection 2 reader."""
+
+import json
+from typing import Dict, Tuple, Type, Any
+
+import attr
+from morecantile import TileMatrixSet
+
+from rio_tiler.constants import WEB_MERCATOR_TMS
+from rio_tiler.errors import InvalidBandName
+from rio_tiler.io import COGReader, MultiBandReader
+
+from ... import get_object
+from ..utils import sceneid_parser
+
+OLI_TIRS_SR_BANDS = (
+    "QA_PIXEL",
+    "QA_RADSAT",
+    "SR_B1",
+    "SR_B2",
+    "SR_B3",
+    "SR_B4",
+    "SR_B5",
+    "SR_B6",
+    "SR_B7",
+    "SR_QA_AEROSOL",
+)
+
+OLI_TIRS_ST_BANDS = (
+    "ST_ATRAN",
+    "ST_B10",
+    "ST_CDIST",
+    "ST_DRAD",
+    "ST_EMIS",
+    "ST_EMSD",
+    "ST_QA",
+    "ST_TRAD",
+    "ST_URAD",
+)
+
+# TODO: ETM and TM appear to have the same level 2 bands? Deduplicate?
+ETM_SR_BANDS = (
+  "QA_PIXEL",
+  "QA_RADSAT",
+  "SR_ATMOS_OPACITY",
+  "SR_B1",
+  "SR_B2",
+  "SR_B3",
+  "SR_B4",
+  "SR_B5",
+  "SR_B7",
+  "SR_CLOUD_QA",
+)
+
+ETM_ST_BANDS = (
+  "QA_PIXEL",
+  "QA_RADSAT",
+  "ST_ATRAN",
+  "ST_B6",
+  "ST_CDIST",
+  "ST_DRAD",
+  "ST_EMIS",
+  "ST_EMSD",
+  "ST_QA",
+  "ST_TRAD",
+  "ST_URAD",
+)
+
+TM_SR_BANDS = (
+  "QA_PIXEL",
+  "QA_RADSAT",
+  "SR_ATMOS_OPACITY",
+  "SR_B1",
+  "SR_B2",
+  "SR_B3",
+  "SR_B4",
+  "SR_B5",
+  "SR_B7",
+  "SR_CLOUD_QA",
+)
+
+TM_ST_BANDS = (
+  "QA_PIXEL",
+  "QA_RADSAT",
+  "ST_ATRAN",
+  "ST_B6",
+  "ST_CDIST",
+  "ST_DRAD",
+  "ST_EMIS",
+  "ST_EMSD",
+  "ST_QA",
+  "ST_TRAD",
+  "ST_URAD",
+)
+
+DEFAULT_L8L1_BANDS = (
```

Comment:
Should this be `OLI_L1_BANDS`?

what about other satellites?

Reply:
Sorry I meant to remove that to focus on Level 2 only in the first PR

Reply:
I think it'll be simpler to split into 2 PRs, one for level 2 (which should be simpler), the second for Level 1